examples/quickstart/quickstart.ipynb

```diff
@@ -326,22 +315,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "metadata": {
     "colab": {
      "base_uri": "https://localhost:8080/"
     },
     "id": "dRzZA406UVVz",
     "outputId": "2d5e9967-87b7-4553-a104-b1b6602e10a6"
    },
-   "outputs": [],
-   "source": [
-    "RUN_NAME_1 = \"standard_mnist_training_run\"\n",
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1;35mCreating run for pipeline: `\u001b[0m\u001b[33;21mmnist_pipeline`\u001b[1;35m\u001b[0m\n",
+      "\u001b[1;35mCache enabled for pipeline `\u001b[0m\u001b[33;21mmnist_pipeline`\u001b[1;35m\u001b[0m\n",
+      "\u001b[1;35mUsing stack `\u001b[0m\u001b[33;21mlocal_stack`\u001b[1;35m to run pipeline `\u001b[0m\u001b[33;21mmnist_pipeline`\u001b[1;35m...\u001b[0m\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mimporter`\u001b[1;35m has started.\u001b[0m\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mimporter`\u001b[1;35m has finished in 0.120s.\u001b[0m\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mdecision_tree_trainer`\u001b[1;35m has started.\u001b[0m\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mdecision_tree_trainer`\u001b[1;35m has finished in 0.065s.\u001b[0m\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mevaluator`\u001b[1;35m has started.\u001b[0m\n",
+      "Test accuracy: 0.7619577308120133\n",
+      "\u001b[1;35mStep `\u001b[0m\u001b[33;21mevaluator`\u001b[1;35m has finished in 0.051s.\u001b[0m\n",
+      "\u001b[1;35mPipeline run `\u001b[0m\u001b[33;21mstandard_mnist_training_run`\u001b[1;35m has finished in 0.250s.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "RUN_NAME_1 = \"decision_tree_mnist_training_run\"\n",
```

Comment:
I think it would make sense to Restart Kernel & Clear Outputs before committing our notebooks. Makes it more readable on github and cleaner for the user.